src/main/java/net/minecraftforge/common/IHorseArmor.java

```diff
@@ -0,0 +1,22 @@
+package net.minecraftforge.common;
+
+import net.minecraft.entity.passive.EntityHorse;
+import net.minecraft.entity.passive.HorseArmorType;
+import net.minecraft.item.ItemStack;
+import net.minecraft.world.World;
+
+public interface IHorseArmor 
+{
+    /** 
```

Comment:
Please write a sentence or two of description here. Even if it is completely redundant with `@return`, it is still good style to have it. [Reference (`@return`)](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@return)